src/test/java/com/redhat/documentation/asciidoc/extraction/model/GitRepositoryTargetTest.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.documentation.asciidoc.extraction.model;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.lib.RepositoryBuilder;
+import org.eclipse.jgit.lib.RepositoryCache;
+import org.eclipse.jgit.transport.RefSpec;
+import org.eclipse.jgit.util.FS;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class GitRepositoryTargetTest {
+    public static final String BRANCH = "test";
+    private static final String REPO_URL = "https://github.com/manaswinidas/Docs-symlink";
```

Comment:
It may be better to make this all self contained and have what is necessary in the tests instead of requiring you to have an internet connection to run the tests.